content/2024/06/maxmind-appoints-seasoned-data-science-leader-rupert-young-as-chief-product-officer.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "MaxMind appoints seasoned data science leader Rupert Young as Chief Product Officer"
+date: "2024-06-11"
+images:
+  - /images/2024/06/rupert_blog.png
+category:
+  - "Company Updates"
+tag:
+  - "News"
+  - "MaxMind team"
```

Comment:
All of our tags and categories on other posts use title casing. Is this intentionally not title cased?

Reply:
not intentional, the only intentional casing is in the byline. I can adjust this one to say MaxMind Team, thanks Dallas!

Reply:
case has been adjusted @dhogan8 thank you!